tests/ProxyManagerTest/Functional/RemoteObjectFunctionalTest.php

```diff
@@ -72,69 +81,84 @@ protected function getJsonRpcAdapter($expectedValue, string $method, array $para
 
     /**
      * @param string|object $instanceOrClassName
-     * @param mixed[]       $params
-     * @param mixed         $expectedValue
+     * @param string $method
+     * @param array $passedParams
+     * @param mixed[] $parametersForProxy
+     * @param mixed $expectedValue
      *
      * @dataProvider getProxyMethods
      *
      * @psalm-template OriginalClass of object
      * @psalm-param class-string<OriginalClass>|OriginalClass $instanceOrClassName
      */
-    public function testXmlRpcMethodCalls($instanceOrClassName, string $method, array $params, $expectedValue) : void
-    {
-        $proxy = (new RemoteObjectFactory($this->getXmlRpcAdapter($expectedValue, $method, $params)))
+    public function testXmlRpcMethodCalls(
+        $instanceOrClassName,
+        string $method,
+        array $passedParams,
+        array $parametersForProxy,
+        $expectedValue
+    ): void {
+        $proxy = (new RemoteObjectFactory($this->getXmlRpcAdapter($expectedValue, $method, $parametersForProxy)))
```

Comment:
Given the context above (and the data provider), let's reword `$parametersForProxy` to `$callParametersExpectedByAdapter`

Reply:
Changed.